File: PHP/7php 3/sum_args.php
<?php
/*
|--------------------------------------------------------------------------
| Sum of all arguments
|--------------------------------------------------------------------------
|
*/

function sum() {
   return array_sum(func_get_args());
}


function sum(...$numbers) {
   $res = 0;
   foreach ($numbers as $n) {
      $res += $n;
   }
   return $res;
}


function sum(...$arr) {
   return array_sum($arr);
}


print_r(sum(1) . ' ' . 1 . '
');
print_r(sum(1, 2) . ' ' . 3 . '
');
print_r(sum(5, 7, 9) . ' ' . 21 . '
');

File: Python/7py 4/sum_args.py
"""
|--------------------------------------------------------------------------
| Sum of all arguments
|--------------------------------------------------------------------------
|
"""

def sum_args(*args):
   return sum(args)


sum_args = lambda *_:sum(_)


def sum_args(*args):
   res = 0
   for i in args:
      res += i
   return res


print(sum_args(1), 1)
print(sum_args(1, 2), 3)
print(sum_args(5, 7, 9), 21)